plugins/modules/query_graphql.py

```diff
@@ -90,22 +90,23 @@
         site_name: den
       query_string: |
         query ($site_name:String!) {
-            sites (name: $site_name) {
+          sites (name: $site_name) {
             id
             name
             region {
                 name
             }
-            }
+          }
         }
 
-  # Get Response with variables
+  # Get Response with variables and set to root keys
   - name: Obtain list of devices at site in variables from Nautobot
     networktocode.nautobot.query_graphql:
       url: http://nautobot.local
       token: thisIsMyToken
       query: "{{ query_string }}"
       variables: "{{ variables }}"
+      populate_root: "yes"
```

Comment:
Shouldn't that be boolean `true` ?

Reply:
In ansible they are equivalent

Reply:
Yes I know, unquoted `yes` is also evaluated as bool `True`.
This is more about documentation consistency, it's described as type bool in `DOCUMENTATION` section.
I was looking form a static-typing perspective, where bool is `True` or `False` without making assumptions about ansible internals, maybe unnecessarily. Anyway I don't have any issue with it, just a friendly suggestion.